ecommerce/tasks.py

```diff
@@ -32,3 +32,10 @@ def send_order_refund_email(order_id):
     order = Order.objects.get(pk=order_id)
 
     send_ecommerce_refund_message(order)
+
+
+@app.task(acks_late=True)
+def process_pending_order_resolutions():
```

Comment:
I don't think we are calling/scheduling this task from anywhere. I'm assuming that we want to schedule this task but I might be wrong. In case, we want to schedule it, is it planned for another PR or we are going to manually run this task as of now?

Reply:
We aren't. We'll need to figure out a timeframe to do it and maybe make some changes - I think we want to leave a pending order alone if it's below an age threshold, since there's potential that the order could be actively in progress (i.e. the learner's filling out the payment form when the task runs). I don't want that discussion to be blocking the rest of it, though, especially since we need to be able to resolve a bunch of pending orders sooner rather than later.

Reply:
Thanks for the clarification, I think it makes sense and I agree that this needs further discussion within a separate thread.